src/tutorials/0006/03.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div>
+        <MultipleChoiceLesson
+        :text="text"
+        :question="question"
+        :choices="choices" />
+    </div>
+</template>
+
+<script>
+import MultipleChoiceLesson from '../../components/MultipleChoiceLesson'
+import text from './03.md'
+
+const question = 'How can we identify the encoding used on specific data when using CIDv1?'
+
+const choices = [
+  {
+    answer: 'These CIDs have a multicodec prefix that specifies which encoding was used.',
+    correct: true,
+    feedback: 'That\'s right!'
+  },
+  {
+    answer: 'All encodings make it safe to try and check the first bytes of the data to figure out which encoding was used',
+    correct: false,
+    feedback: 'Hmm, not always. Please check the information above'
+  },
+  {
+    answer: 'We can\'t know for sure, but the multicodec prefix included in the CID is a hint that is not always guaranteed to be right.',
+    correct: false,
```

Comment:
This is actually true, there's no guarantee when you get the data that it was encoded using the IPLD format in the CID.

Reply:
True, in theory might be possible. Will change this

Reply:
Changed to `The codec used on CIDv1 is always "base58btc".`